src/tag/index.ts

```diff
@@ -3,4 +3,4 @@ export * from "./tag.component";
 export * from "./tag-icon.directive";
 export * from "./tag-selectable.component";
 export * from "./tag-operational.component";
-export * from "./tag.module";
+// export * from "./tag.module";
```

Comment:
```suggestion
export * from "./tag.module";
```

We need to export from index to make it easier to import module